spec/spec.md

```diff
@@ -1314,6 +1314,15 @@ processing steps. Additional verification of credential data or subsequent
 validation required by a given Verifier are left to the Verifier's systems, code 
 and business processes to define and execute.
 
+During validation, each Input Descriptor Object ****MUST**** refer to only a
+singular presentation within a _Presentation Submission_, such that all checks
```

Comment:
should presentation here be container? I think the terminology is confusing. On one hand a presentation submission contains a presentation, but the understanding here is that it contains multiple presentations.

Reply:
Yeah, this kinda highlights the importance of #90. I couldn't find a good word to express "things protected by the same signature within the same hierarchical grouping". Let's avoid "presentation" due to its overloading, I'll use "container" only. Does that sound reasonable?

Reply:
sounds reasonable. we can always amend after #90 is settled.

Reply:
Please let me know if the latest commit fixes your concern! Thanks